h/views/api_auth.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.auth.services import TOKEN_TTL
+from h.exceptions import OAuthTokenError
+from h.util.view import json_view
+
+
+@json_view(route_name='token', request_method='POST')
```

Comment:
It currently has to be a POST form request, yes. This can either be `application/x-www-form-urlencoded` or `multipart/form-data`.
Is it worth supporting JSON POST bodies as well? I'm not sure (I'm not against it, but I rather don't implement it if it isn't needed).
Maybe @sean-roberts has some input on this?

_I'm sure you already know this, but a request to `/api/token` does not need the developer token (because you do include it in your example)._